tests/test_sanitise_text.py

```diff
@@ -3,12 +3,16 @@
 from notifications_utils.sanitise_text import SanitiseText, SanitiseSMS, SanitiseASCII
 
 
+@pytest.mark.parametrize("chars, cls", [("ÀÂâçÈÊêËëÎîÏïÔôŒœÙÛû₣", SanitiseSMS)])
+def test_encode_chars_sms_fr_not_downgraded(chars, cls):
+    for char in chars:
+        assert cls.encode_char(char) == char
```

Comment:
So are we opting to allow non-GSM french accented characters but without downgrading them, which results in more SMS parts?

I am not challenging the decision but just want to know which way product is going.

Reply:
Code looks good to me. I agree, we should have a discussion so everyone is aware of the tradeoffs here.